spec/models/marc_fields/linked_serials_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'spec_helper'
+
+describe LinkedSerials do
+  include MarcMetadataFixtures
+  subject { described_class.new(SolrDocument.new(marcxml: marc)) }
+
+  describe 'labels' do
+    context '780s' do
+      let(:marc) { title_change_fixture }
+
+      it 'returns a value for each 780' do
+        expect(subject.values.length).to eq 2
+      end
+
+      it 'has a label based on the indicator2' do
+        expect(subject.values.first[:label]).to eq 'Continues'
+        expect(subject.values.last[:label]).to eq 'Absorbed in part by'
+      end
+    end
+
+    context '785' do
+      let(:marc) { merged_with_serial_fixture }
+
+      it 'returns the "Merged with" label for the first merged with 785' do
+        expect(subject.values.first[:label]).to eq 'Merged with'
+      end
+
+      it 'returns the "and with" label middle merged with 785s' do
+        expect(subject.values[1][:label]).to eq 'and with'
+      end
+
+      it 'returns the "Merged with" label for the last merged with 785' do
+        expect(subject.values.last[:label]).to eq 'to form'
+      end
+    end
+
+    context 'other serials' do
+      let(:marc) { main_entry_and_title_serial_fixture }
+
+      it 'has a field based label' do
+        expect(subject.values.last[:label]).to eq 'Constituent unit'
+      end
+    end
+  end
+
+  describe 'values' do
+    context 'when main entry ($a) is present' do
+      context 'when title data ($s or $t) is present' do
+        let(:marc) { main_entry_and_title_serial_fixture }
+
+        it 'included a hash that combines the main entry with the title and indicated author_title search' do
```

Comment:
Is this another way of saying " If $s AND $t are present, choose $s"  from #1121 ?    I confess confusion.